File: Producto/backend/src/main/java/com/proyecto/macrofit/rutinaejercicio/model/RutinaEjercicio.java
package com.proyecto.macrofit.rutinaejercicio.model;

import lombok.Data;

@Data
public class RutinaEjercicio {

    private Integer idRutinaEjercicio;
    private Integer idEjercicio;
    private Integer idRutina;
    private Integer dia;
    private Integer orden;
    private Integer series;
    private Integer tiempoSeg;
    private Integer repeticiones;
    private Float pesoReferencia;

}

File: Producto/MacroFit/app/src/main/java/com/duoc/macrofit/rutinas/model/RutinaEjercicio.kt
package com.duoc.macrofit.rutinas.model

import com.google.gson.annotations.SerializedName

data class RutinaEjercicio(
    val idRutinaEjercicio: Int,

    val idEjercicio: Int,

    val idRutina: Int,

    val dia: Int? = 1,
    val orden: Int? = 1,

    val series: Int?,
    val repeticiones: Int?,
    val tiempoSeg: Int?,
    val pesoReferencia: Float?
)